editor/src/clj/editor/gl/vertex2.clj

```diff
@@ -118,16 +62,19 @@
 
 (defn wrap-vertex-buffer
   [vertex-description usage ^Buffer buffer]
+  {:pre [(graphics.types/vertex-description? vertex-description)
+         (graphics.types/usage? usage)
+         (instance? Buffer buffer)]}
   (let [buffer-items-per-vertex (buffer-items-per-vertex buffer vertex-description)]
     (->VertexBuffer vertex-description usage buffer buffer-items-per-vertex 0)))
 
 (defn wrap-buf
   ^ByteBuffer [^bytes byte-array]
-  (buffers/wrap-byte-array byte-array :byte-order/little-endian))
+  (buffers/wrap-byte-array byte-array :byte-order/native))
 
 (defn make-buf
   ^ByteBuffer [byte-capacity]
-  (buffers/new-byte-buffer byte-capacity :byte-order/little-endian))
+  (buffers/new-byte-buffer byte-capacity :byte-order/native))
```

Comment:
More places where we assumed little-endian architecture. Again, this change doesn't really change anything since we're only building the editor for big-endian.